docs/source/science/calculators/morphology.rst

```diff
@@ -4,8 +4,200 @@ Morphology
 ==========
 
 
-Description of the Morphology Calculator.
+The **morphology** calculator represents is the first part of the **mobility** calculator workflow (:ref:`science_calculators_mobility`), which terminates after the morphology is generated and analyzed.
+
+.. list-table:: Morphology Workflow Overview
+   :widths: 30 30 30
+   :header-rows: 1
+
+   * - **Nanomatch Software**
+     - **Scientific Role**
+     - **Illustration**
+   * - `Parametrizer <http://docs.nanomatch.de/nanomatch-modules/Parametrizer/Parametrizer.html>`_
+     - | Geometry optimization
+     - .. image:: mobility/parametrizer.png
+          :width: 300px
+          :align: center
+   * - `DihedralParametrizer <http://docs.nanomatch.de/nanomatch-modules/DihedralParametrizer/DihedralParametrizer.html>`_
+     - | Computation of intramolecular
+       | forcefields
+     - .. image:: mobility/dhp.png
+          :width: 300px
+          :align: center
+   * - `Deposit <http://docs.nanomatch.de/nanomatch-modules/Deposit/Deposit.html>`_
+     - | Simulation of the physical
+       | vapor deposition (PVD)
+       | to obtain atomistic morphology
+     - .. image:: mobility/deposit.png
+          :width: 300px
+          :align: center
+
+Implemented Scientific Methods
+------------------------------
+
+The following scientific methods are implemented in the Calculator "**morphology**":
+
+Molecular Structure Optimization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*obabel*, *xtb*, and *Density Functional Theory (DFT)* are used to generate initial 3D conformers, pre-optimize and optimize the geometry, and compute partial charges via an electrostatic potential (ESP) fit for single molecules in a vacuum. DFT optimization is performed with def2-TZVP/B3LYP level of theory.
+
+Morphology Generation
+~~~~~~~~~~~~~~~~~~~~~
+
+The *DEPOSIT protocol* [1]_ simulates physical vapor deposition to generate thin-film morphologies with atomistic resolution. This involves Monte Carlo (MC) based basin hopping with simulated annealing (SA) to model intermolecular interactions during deposition. In total, 1000 molecules are deposited into a box with a base size of 100x100 Å.
+
+
+Output
+------
+
+This Calculator provides the following properties:
+
+- HOMO and LUMO (see :ref:`science_properties_HOMOLUMO`)
+- Dipole (property not yet described / available)
+- Morphology and its related properties (see section Files).
```

Comment:
This should link to the Morphology property

Reply:
linked